content/neuvector-prime/azure/contents.lr

```diff
@@ -426,16 +426,19 @@ The AKS cluster already has the extension. Uninstall the extension as it suggest
    !["Uninstall extension"](./uninstall.png)
 
 
-### Checking the Kubernetes Service Extension version:
+###Upgrading a NeuVector Prime PAYG Cluster
 
-To see the most recent published extension versions, execute the following command in Cloud Shell.
+The existing AKS cluster must be on a supported PAYG platform with its Neuvector configuration. To manually upgrade the recent version via the "az k8s-extension update" command.
```

Comment:
I don't understand what this is trying to say...  Is this saying that this upgrade only works if they're upgrading an existing NV PAYG deployment?
The second sentence needs to be reworded a bit as well... maybe the following:
Manually upgrading the version of NeuVector will utilize the "az k8s-extension update" command.

Reply:
it was copy from the old upgrade aws doc. Here is the updated

`o Update the NeuVector version to the latest version supported by the marketplace listing, via the "az k8s-extension update" command.              

Execute the following commands in Cluster Cloud Shell:

-  ---
   **NOTE** if cluster is deployed in westcentralus location, execute the following command.
           This command will also work on other regions once the feature is rolled out to other regions.

-  ---

`